### ISSUE TITLE
Moons units

### ISSUE THREAD

**Issue by hldouglas** [hldouglas/dsn1675-planet-website#2]
I want to have the description list in moons.index to left align under the image. I attempted to make another grid and unit within the already grid unit nest, but it still aligns with the paragraphs. Should I make the descriptions outside of the grid and unit nest?

**Comment by hldouglas** [comment]
@thomasjbradley

**Comment by thomasjbradley** [comment]
Yeah, the description list needs to be outside the inner grid `<div>` but still inside the outer one, like this:

``` html
<div class="grid">
  <div class="unit">

    <div class="grid">
    </div>

    <dl>
    </dl>
⋮
```
